### DISCUSSION TITLE
Assassin's Creed The Ezio Collection

### DISCUSSION THREAD

**Discussion by Nihalshahazi123** [skyline-emu/skyline#689] [Game Requests]
I would really appreciate if you tried to make these games playable the thought of playing assassin's creed 2 brother hood and revaluations on my phone and also GTA definite edition because it might be easier for the phone to run thank you for reading

**Comment by PixelyIon** [comment]
I'll be curbing this request to just "Assassin's Creed The Ezio Collection" as we already have a request for the latter at #511.
